### DISCUSSION TITLE
专业版支付完成

### DISCUSSION THREAD

**Discussion by RoshanQ666** [getlantern/lantern-forum-cn#604] [General]
请帮忙确认下账户状态

**Comment by MaxMKMax** [comment]
@RoshanQ666 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by MaxMKMax** [comment]
@RoshanQ666 谢谢您支持蓝灯. 好的, 我帮您提交问题. 请您过几天查看这个帖子或邮箱. 我们会给您发激活码.

**Comment by RoshanQ666** [comment]
Thanks for the helps~
________________________________
发件人: Mike K ***@***.***>
发送时间: 2024年6月18日 22:23
收件人: getlantern/lantern-forum-cn ***@***.***>
抄送: RoshanQ666 ***@***.***>; Mention ***@***.***>
主题: Re: [getlantern/lantern-forum-cn] 专业版支付完成，未显示续期 (Discussion #604)


@RoshanQ666<https://github.com/RoshanQ666> 谢谢您支持蓝灯. 好的, 我帮您提交问题. 请您过几天查看这个帖子或邮箱. 我们会给您发激活码.

―

**Comment by RoshanQ666** [comment]
Hi Mike
pls refer to the ticket number 73680187
thanks.
________________________________
发件人: Mike K ***@***.***>
发送时间: 2024年6月18日 22:18
收件人: getlantern/lantern-forum-cn ***@***.***>
抄送: RoshanQ666 ***@***.***>; Mention ***@***.***>
主题: Re: [getlantern/lantern-forum-cn] 专业版支付完成，未显示续期 (Discussion #604)


@RoshanQ666<https://github.com/RoshanQ666> 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

―
